titanic/pipeline.ipynb

```diff
@@ -34,15 +34,15 @@
     "    description='sample titanic pipeline with dkube components'\n",
     ")\n",
     "def titanic_pipeline():\n",
-    "    train       = dkube_training_op(container = '{\"image\":\"ocdr/d3-datascience-sklearn:v0.23.2-1\"}',\n",
+    "    train       = dkube_training_op(auth_token = token,container = '{\"image\":\"ocdr/d3-datascience-sklearn:v0.23.2-1\"}',\n",
```

Comment:
All pipelines should follow the convention of using token as pipeline argument..